package.json

```diff
@@ -42,7 +42,6 @@
     "eslint-formatter-todo": "^1.0.1",
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-jsx-a11y": "6.7.1",
-    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "^3.0.3"
```

Comment:
Can prettier be a devDependency now? I don't think we're using it to format the code in this repo, so maybe we could just remove it altogether.

Reply:
Yeah I was wondering about that too. Let's move to dev deps